### ISSUE TITLE
Swing `]` end of list character under other entries

### ISSUE THREAD

**Issue by seagreen** [chrisdone/hindent#247]
The style guide [suggests](https://github.com/tibbe/haskell-style-guide/blob/master/haskell-style.md#list-declarations) that a list ending `]` goes under the last entry:

```
exceptions =
    [ InvalidStatusCode
    , MissingContentHeader
    , InternalServerError
    ]
```

But when I try this input on the [website](http://chrisdone.com/hindent/):

```
exceptions =
    [ InvalidStatusCode
    , MissingContentHeader
    , InternalServerError
    , SomeOtherException
    ]
```

It outputs:
```
exceptions =
    [ InvalidStatusCode
    , MissingContentHeader
    , InternalServerError
    , SomeOtherException]
```

**Comment by seagreen** [comment]
PS Super excited to see the haskell community moving closer to a `gofmt` like way of doing things. Also I voted for 4-spaces in the poll but don't care as long as there's a standard:)

**Comment by tfausak** [comment]
This is a duplicate of #228. I fixed it in #240, which I assume hasn't been deployed to the website yet.

**Comment by seagreen** [comment]
Sweet, sorry for the dupe.
